public/components/stub-modal/stub-modal.html

```diff
@@ -50,9 +50,31 @@ <h4 class="modal-title">{{ modalTitle }}</h4>
                 <div ng-if="cdesks.length === 0">
                     <p class="help-block">Selecting commissioning info is temporarily unavailable</p>
                 </div>
-                <label for="commissionedLength">Commissioned Length</label>
+                <label for="commissionedLength">Word Count</label>
+                <a class="feedback-link"  href="https://docs.google.com/forms/d/e/1FAIpQLSeZje55T3OnErlTI_8iGuyZERjDy2Pybh8fdPmbnjy1PNFDAw/viewform" target="_blank">
+                    <span class="feedback-title" title="Send Feedback">Send Feedback</span>
+                </a>
+                <div class="word-count-buttons">
+                    <button class="btn btn-default btn-sm" ng-model="stub.isBreakingNews" btn-radio="true" ng-click="resetWordCount()">Breaking News</button>
+                    <button
+                        ng:repeat="wordCountSuggestion in wordCountSuggestions"
+                        class="btn btn-default btn-sm word-count-suggestion"
+                        ng-model="stub.commissionedLength"
+                        btn-radio="wordCountSuggestion"
+                        ng-click="resetIsBreakingNews()"
+                    >
+                        {{ wordCountSuggestion }}
+                    </button>
```

Comment:
There is a slight juggling of two different types of button here to get this to work. It's clunky, but allows us to have radio buttons without any default (linked to input state).

`btn-radio` is an old Angular-Bootstrap concept.